OutOfSchool/OutOfSchool.DataAccess/Repository/IParentRepository.cs

```diff
@@ -1,8 +1,16 @@
-﻿using OutOfSchool.Services.Models;
+﻿using System;
+using System.Collections.Generic;
+using OutOfSchool.Services.Models;
 
 namespace OutOfSchool.Services.Repository
 {
     public interface IParentRepository : IEntityRepository<Parent>
     {
+        /// <summary>
+        /// Find user onformation by parent Id.
+        /// </summary>
+        /// <param name="parents">Parent Ids.</param>
+        /// <returns>Anonymous Type which contains part of user information.</returns>
+        public IEnumerable<Tuple<long, string, string>> GetUsersByParents(IEnumerable<long> parents);
```

Comment:
Generic thought about the whole PR:
`GetUsersByParents` looks like central idea of feature implementation and its returning type `Tuple<long, string, string>` for me sounds like something vague. Number and 2 strings, what they stand for? "Anonymous Type which contains part of user information" - bit unclear, you know.

From the usage (`RatingService.GetUsersAsync()`):
```
var newUsers = parentRepository.GetUsersByParents(...);
...
var userInfo = newUsers.FirstOrDefault(p => p.Item1 == ratingDtosList[i].ParentId);
ratingDtosList[i].FirstName = userInfo?.Item2;
ratingDtosList[i].LastName = userInfo?.Item3;
```
i see that we expect something like (pseudocode) `Dictionary<parentId, List<user>>` or even `Dictionary<parentId, user>`  (because of `FirstOrDefault()`).

Why not to return such shape that is more meaningful? 

`public Dictionary<long, UserDto> GetSomeUserByParentIds(IEnumerable<long> parentIds);`

or

`public Dictionary<long, List<UserDto>> GetUsersByParentIds(IEnumerable<long> parentIds);`

or 

`public Lookup<long, UserDto> GetUsersByParentIds(IEnumerable<long> parentIds);`

And write method implementation accordingly.

Reply:
Please try to avoid using tuples as a return type in the public methods - it does not actually clear what are those strings actually stands for.
Please take a look at the [ValueTuples](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples).

Reply:
Dear @VadymLevkovskyi and @h4wk13,
Thank you a lot fot these advise and that you describet it so great.
I have decide that to add aliases for tuples will be better because I think no need to add Dto into repository.
Thank you.

Reply:
Could you please describe why it is better to use tuples, especially Tuple class instead of value tuples ?

Reply:
Dear @h4wk13,
I have to rewrite my code to value tuples according to your recommendation.
Please check.
Thank you.